### PR TITLE
Update content of vertical navigation responsive state

### PR DESCRIPTION
**Pull request by deernano**
## Description
*Update the responsive state of vertical navigation
* PR: https://github.com/patternfly/patternfly-design/issues/411
  Jira story: https://patternfly.atlassian.net/browse/PTNFLY-2581

## Changes

Changed the design.md and img files
* Change 01 Update pictures in img file
* Change 02 Delete useless pictures in img file
* Change 03 Update the content of design.md

## Contribution Submission Checklist

- [X] A GitHub issue was created and reviewed with the PatternFly community.
- [X] The pattern design overview.md and design.md templates were used to provide all information, relevant to this design.
- [X] Any new files have been added to the folder for that pattern.
- [X] The folder, for brand new patterns, has been titled with the name of the pattern.
- [X] All images are in an "img" folder and embedded images have the correct filepath.
- [X] The overview.md, design.md and site.md files, for this design have been commited and pushed to the patternfly-design repository, for review.

### CODE REVIEW

File: pattern-library/navigation/vertical-navigation/design/design.md [hunk 1]
```diff
@@ -55,30 +55,30 @@ Tertiary navigation is non-persistent and only appears on hover. The tertiary na
...
 
-![navigation-vertical-notifications-responsive-callout](img/navigation-vertical-notifications-responsive-callout.png)
-
-1. **Back Link**: Clicking the back link restores the primary-level menu.
-2. **Notification Drawer**: The notification drawer takes the entire screen's width when on small screens.)
--->
+### Vertical Navigation with Notification Drawer | Responsive State
+![navigation-vertical-responsive5](img/navigation-vertical-responsive5.png)
+1. **Bell Icon**: The icon of notification drawer is exposed on the second line of the masthead in the responsive state.
+2. **Notification Drawer**: When clicking the Icon, the notification drawer will slide out from the right edge and the drawer will take over the whole screen width. Click “X” button on the right edge to close the notification drawer.
```
Comment (mcarrano) [inline pattern-library/navigation/vertical-navigation/design/design.md]: Please reference the Notification Drawer pattern where this is described in more detail (http://www.patternfly.org/pattern-library/communication/notification-drawer/#/design)

File: pattern-library/navigation/vertical-navigation/design/design.md [hunk 2]
```diff
@@ -55,30 +55,30 @@ Tertiary navigation is non-persistent and only appears on hover. The tertiary na
...
-- **Mobile** (less than 768 px): Navigation is hidden under “hamburger” icon.
+
+![navigation-vertical-responsive2](img/navigation-vertical-responsive2.png)
+- **Mobile** (less than 768 px): The masthead utility items and navbar will be hidden under “hamburger” icon. The logotype and the “hamburger” icon will stack into two line. The Product logo will be hidden only remain the product name. The product name will collapse on scroll.
+
 
 ### Primary Navigation | Responsive State
 
-![navigation-vertical-responsive-state-callout](img/navigation-vertical-responsive-callout.png)
+![navigation-vertical-responsive3](img/navigation-vertical-responsive3@2x.png)
```
Comment (mcarrano) [inline pattern-library/navigation/vertical-navigation/design/design.md]: The scale of this image seems to be off for some reason.

**Comment by mcarrano** [comment]
@serenamarie125 @catrobson @maryclarke @jeperry @LHinson @kybaker This PR updates the vertical appearance of the masthead in the response state to expose branding as we discussed earlier. Please review.

**Comment by mcarrano** [comment]
@deernano For my comment regarding the responsive state with the notification drawer, I think that I was just trying to say that we should not duplicate the behaviors of the notification drawer here. The reader can just link to the [Notification Drawer pattern](http://www.patternfly.org/pattern-library/communication/notification-drawer/#/design) if they want that information. That avoid the risk that that these become out of sync.  But what we should do is to also update the mockups on the notification drawer page so the mockups match the new responsive masthead approach.  Does that make sense?

**Comment by deernano** [comment]
@mcarrano Please reivew the new commit . Thank you~

**Comment by jeff-phillips-18** [comment]
How do changes like this make it into the implementation repositories? Do you plan to add issues to each implementing repository to make these updates?

@LHinson @serenamarie125

**Comment by LHinson** [comment]
@jeff-phillips-18 That's a good question. We could improve our coordination across the repos.... This also seems related to the issue that if something changes in core, how do we best inform the js frameworks that depend on core? Let me schedule a repo leads meeting and add this as an agenda item for discussion and brainstorming on how we can solve this. There is some other topics that we need to cover so it seems like a good time to chat.

**Comment by mcarrano** [comment]
@LHinson Can you give this a quick review and go ahead and merge if you are comfortable with these changes.

**Comment by mcarrano** [comment]
I am going to merge as this has been sitting in the queue for a while.

**Pull request merged**